pkg/codegen/schema_test.go

```diff
@@ -213,3 +213,243 @@ func TestProperty_GoTypeDef(t *testing.T) {
 		})
 	}
 }
+
+func TestPropertyNullable_GoTypeDef(t *testing.T) {
```

Comment:
This test should be collapsed in with the above, no? At the very least:

```suggestion
func TestProperty_GoTypeDef_nullable(t *testing.T) {
```

To indicate what it's testing